zcash_client_backend/CHANGELOG.md

```diff
@@ -105,6 +109,15 @@ and this library adheres to Rust's notion of
 
 ### Removed
 - `zcash_client_backend::data_api`:
+  - `AccountBalance::unshielded`. `AccountBalance` no longer provides the `unshielded`
+    method returning a `NonNegativeAmount` for the total of transparent funds.
```

Comment:
This is not removed, only deprecated. Move it to the Deprecated section with the wording: "`AccountBalance::unshielded`. Instead use `unshielded_balance` which provides a `Balance` value. Its `total()` method can be used to obtain the total of transparent funds."